explorer/files/ncpi-catalog/build-ncpi-catalog.ts

```diff
@@ -68,6 +70,7 @@ export async function buildNCPIPlatformStudies(
   platformStudyStubs: NCPIPlatformStudyStub[]
 ): Promise<NCPIStudy[]> {
   const ncpiStudies: NCPIStudy[] = [];
+  const studiesById: NCPIStudyDict = {};
```

Comment:
Hi @hunterckx Thanks for this. 0Looking good. 

Would you be able to make studies by ID a Map<string, NCPIStudy> ? 


Thanks!, D